### ISSUE TITLE
Allow markdown panic guides

### ISSUE THREAD

**Issue by lucas42** [Financial-Times/go-fthealth#9]
Currently, go-fthealth expects the panicGuide field to be a URL, but [the spec says](https://docs.google.com/document/edit?id=1ftlkDj1SUXvKvKJGvoMoF1GnSUInCNPnNGomqTpJaFk#heading=h.3zknx8k6ja6i) markdown is supported.  We should handle markdown correctly when displaying it to users, rather than treating it as a URL.

Example use case: https://github.com/Financial-Times/coreos-version-checker/blob/master/version.go#L26

**Comment by tosan88** [comment]
Is this still valid or is it resolved as per https://github.com/Financial-Times/go-fthealth/pull/10?

**Comment by lucas42** [comment]
No, #10 will show plaintext copy for panic guides, but doesn't handle markdown.
